helix-core/src/main/java/org/apache/helix/monitoring/metrics/WagedRebalancerMetricCollector.java

```diff
@@ -38,7 +41,11 @@
 
     // The following latency metrics are related to AssignmentMetadataStore
     StateReadLatencyGauge,
-    StateWriteLatencyGauge
+    StateWriteLatencyGauge,
+
+    // Count of any rebalance failure.
+    // Note the rebalancer may still be able to return an assignment based on the previous record on an error.
```

Comment:
This line of comment is longer than 100 chars.

Reply:
The format file does not care about the comments, I guess. Is it configurable?
I will fix it manually this time.